test/fixtures/profiles/profile-with-gem-dependency/controls/example.rb

```diff
@@ -0,0 +1,11 @@
+require "money"
```

Comment:
Looks like we need to implement something for `inspec check`, too:
```
[cwolfe@lodi inspec-5]$ be inspec check test/fixtures/profiles/profile-with-gem-dependency
bundler: failed to load command: inspec (/Users/cwolfe/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bin/inspec)
LoadError: cannot load such file -- money
  /Users/cwolfe/sandbox/inspec/inspec-5/lib/inspec/dsl_shared.rb:33:in `require'
  /Users/cwolfe/sandbox/inspec/inspec-5/lib/inspec/dsl_shared.rb:33:in `require'
```

Reply:
Fixed this. Thanks, @clintoncwolfe !!